index.html

```diff
@@ -1,12 +1,35 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Js Girls - Base application</title>
-
-    <link href="styles/vendor/bootstrap.min.css" rel="stylesheet">
+    <title>Js Girls - Awesome Tasks</title>
+    <link href="styles/app.css" rel="stylesheet">
+    <script id="item-template" type="text/template">
+      <li class="_CLASS_">
+        <div class="checkbox">
+          <label>
+            <input type="checkbox" onclick="toggleStatus(event)" _CHECKED_>
+            <span class="item-text">_TEXT_</span>
+          </label>
+          <button class="close" onclick="removeItem(event)">&times;</button>
+        </div>
+      </li>
+    </script>
   </head>
   <body>
-    <h1>JS Girls</h1>
+    <h2>Awesome Tasks</h2>
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <input id="new-item" class="form-control" placeholder="Add a new task..." onkeyup="createNew(event)" autofocus="autofocus">
+      </div>
+      <ul id="task-list" class="list-group"></ul>
+      <div class="panel-footer">
+        <button class="btn btn-link btn-xs" onclick="filterItems()">All</button>
+        <button class="btn btn-link btn-xs" onclick="filterItems('active')">Active</button>
+        <button class="btn btn-link btn-xs" onclick="filterItems('completed')">Completed</button>
+        <button class="btn btn-link btn-xs pull-right" onclick="clearCompleted()">Clear completed</button>
```

Comment:
Four buttons with same classes. Couldnt we have a `div.panel-footer button` style to simplify the code?